src/main/java/base/UserProfile.java

```diff
@@ -0,0 +1,14 @@
+package base;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+public class UserProfile {
+
+    @Getter
```

Comment:
we can move this annotation before AllArgsConstructor

Reply:
Not working without @Getter annotation.

Reply:
I understood. I must move @Getter before @AllArgsConstructor annotation.